### ISSUE TITLE
Remove Metadata Information Package

### ISSUE THREAD

**Issue by GuiltyDolphin** [duckduckgo/duckduckgo#180]
Seeing as we've removed metadata from the Instant Answers, maybe we should remove the `DDG::Meta::Information` package to prevent anyone from adding the metadata in the future?

/cc @zachthompson @moollaza

**Comment by zachthompson** [comment]
Yes, this is on the radar.  We have some things that still depend on it and need to be updated.
